lighthouse-core/test/audits/third-party-summary-test.js

```diff
@@ -10,6 +10,8 @@ const networkRecordsToDevtoolsLog = require('../network-records-to-devtools-log.
 
 const pwaTrace = require('../fixtures/traces/progressive-app-m60.json');
 const pwaDevtoolsLog = require('../fixtures/traces/progressive-app-m60.devtools.log.json');
+const netlifyTrace = require('../fixtures/traces/progressive-app-m60-netlify.json');
```

Comment:
I don't think we need to bring a whole trace in to test this :)

should be able to use `networkRecordsToDevtoolsLog` like the "be not applicable when no third parties are present" test does with some specific URLs. I think it would help make the explicit case we're trying to catch a little more obvious too